### ISSUE TITLE
Disabling Gatekeeper leads to a _less_ secure laptop

### ISSUE THREAD

**Issue by simonmaddox** [o2Labs/puppet-security#1]
The readme in this repo is titled "A More Secure laptop", but you're disabling an important security feature in OS X.

**Comment by simonmaddox** [comment]
@kevinprince and this is why you should leave Gatekeeper enabled: https://developer.apple.com/news/?id=09222015a

**Comment by kevinprince** [comment]
yeah, their was reasoning at the time. Have actually changed view on this
now :)
